.drone.star

```diff
@@ -955,7 +959,9 @@ def localApiTests(suite, storage, extra_environment = {}):
         "image": OC_CI_PHP % DEFAULT_PHP_VERSION,
         "environment": environment,
         "commands": [
-            "make test-acceptance-api",
+            # merge the expected failures
+            "" if get_remotephp_config(ctx) else "cat %s/expected-failures-without-remotephp.md >> %s" % (test_dir, expected_failures_file),
```

Comment:
I tried to move this command to the makefile but that will change the files (merge one file to another). So, I leave this as it is for now